apps/worker/test_utils/base.py

```diff
@@ -16,9 +15,7 @@ def convert_report_to_better_readable(self, report):
             file_report = report.get(filename)
             lines = []
             for line_number, line in file_report.lines:
-                (coverage, line_type, sessions, messages, complexity) = (
-                    dataclasses.astuple(line)
-                )
+                (coverage, line_type, sessions, messages, complexity) = line.astuple()
```

Comment:
the description says messages was removed?...

Reply:
hum looks like they were not _completely_ removed, but replaced with a constant, empty value. Makes sense to keep the interface I guess.

Reply:
yes, because `complexity` is still used (although I believe we should remove that at some point as well), and we have to keep the order the same.